src/CosmosDB/Microsoft.Azure.Workflows.ServiceProvider.Extensions.CosmosDB.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
do we need a separate sln file for each proj - can we just have one root level one?